SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -164,19 +170,21 @@ void ASpatialDebugger::OnAuthorityGained()
 			WorkerRegions.SetNum(LBStrategyRegions.Num());
 			for (int i = 0; i < LBStrategyRegions.Num(); i++)
 			{
-				const TPair<VirtualWorkerId, FBox2D>& LBStrategyRegion = LBStrategyRegions[i];
-				const PhysicalWorkerName* WorkerName =
-					NetDriver->VirtualWorkerTranslator->GetPhysicalWorkerForVirtualWorker(LBStrategyRegion.Key);
 				FWorkerRegionInfo WorkerRegionInfo;
-				WorkerRegionInfo.Color = (WorkerName == nullptr) ? InvalidServerTintColor : SpatialGDK::GetColorForWorkerName(*WorkerName);
+				const TPair<VirtualWorkerId, FBox2D>& LBStrategyRegion = LBStrategyRegions[i];
+				WorkerRegionInfo.VirtualWorkerID = LBStrategyRegion.Key;
+				WorkerRegionInfo.WorkerName = *NetDriver->VirtualWorkerTranslator->GetPhysicalWorkerForVirtualWorker(LBStrategyRegion.Key);
```

Comment:
Previously we had a nullptr check, whereas here we're dereferencing without checking - are we confident there's no risk here?

Reply:
Good point. I will put the nullptr check back.